src/java/picard/analysis/CompareMetrics.java

```diff
@@ -17,13 +17,22 @@
  * Compare two metrics files.
  */
 @CommandLineProgramProperties(
-        usage = CompareMetrics.USAGE,
-        usageShort = CompareMetrics.USAGE,
+        usage = CompareMetrics.USAGE_SUMMARY + CompareMetrics.USAGE_DETAIL,
+        usageShort = CompareMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CompareMetrics extends CommandLineProgram {
-
-    static final String USAGE = "Compare two metrics files";
+    static final String USAGE_SUMMARY =  "Compares two metrics files";
+    static final String USAGE_DETAIL = "Compares the headers of the two input metrics files that have the same structure," +
+            " but come from different underlying metric classes.  Outputs can be either equal" +
```

Comment:
Ah, yup, see the line 

    log.info("Files " + metricsFiles.get(0) + " and " + metricsFiles.get(1) + "are " + status);

That is the instruction to output the result to the info-level log ("log.info")

Yep, please do ask on hipchat. If you don't get an answer quickly, open an issue in dsde-docs and we'l try to attract attention to that.